app/views/devise/passwords/new.html.erb

```diff
@@ -4,15 +4,14 @@
   <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
     <%= render "devise/shared/error_messages", resource: resource %>
 
-    <div class="field grid-field">
-      <%= f.label :email %>
-      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    <div class="icon-wrapper email-field">
+      <%= f.email_field :email, class: 'text-field', placeholder: 'Email', maxlength: '40' %>
     </div>
 
-    <div class="actions submit-button flex-column-centred">
-      <%= f.submit "Send me reset password instructions" %>
-    </div>
-  <% end %>
+    <div id='user_submit' class="general-button glaring">Send me reset password instructions</div>
```

Comment:
Need to correct format of error messages - e.g. the error shown when an invalid email is submitted here